src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestWithAcquireLease.java

```diff
@@ -66,20 +66,19 @@ public void should_process_zip_file_only_once_by_processor_thread_which_acquires
 
         // Then
         // Only one thread should be able to acquire lease and others should fail
-        assertThat(
-            StringUtils.countMatches(
-                outputCapture.toString(),
-                "Lease already acquired for container test and zip file 10_24-06-2018-00-00-00.zip")
-        ).isEqualTo(4);
 
         // We expect only one envelope which was uploaded and other failed
         List<Envelope> envelopes = envelopeRepository.findAll();
         assertThat(envelopes.size()).isEqualTo(1);
 
         // and
-        // We expect only two events one for doc upload and another for doc processed
+        // We expect only 3 events: doc upload, doc processed and notification sent
```

Comment:
imho comment no longer needed now when the test explicitly checks events (as opposed to just the number of them previously)